src/EnergyPlus/ChillerExhaustAbsorption.cc

```diff
@@ -2270,4 +2254,8 @@ void ExhaustAbsorberSpecs::updateHeatRecords(EnergyPlusData &state, Real64 MyLoa
     this->HeatElectricEnergy = this->HeatElectricPower * RptConstant;
 }
 
+void ExhaustAbsorberSpecs::oneTimeInit([[maybe_unused]] EnergyPlusData &state)
+{
+}
```

Comment:
Correct me if I'm wrong, but this will eventually go away once all of the `oneTimeInits` have been moved to member methods of `PlantComponent`, right?